server/handlers/project_command_output_handler.go

```diff
@@ -167,11 +167,6 @@ func (p *AsyncProjectCommandOutputHandler) writeLogLine(pull string, line string
 	}
 	p.receiverBuffersLock.Unlock()
 
-	// No need to write to projectOutputBuffers if clear msg.
-	if line == models.LogStreamingClearMsg {
-		return
-	}
-
```

Comment:
Do we want to write ClearMsg to the `projectOutputBuffers`?

Reply:
I reverted it